lib/lockstep_sdk/lockstep_api.rb

```diff
@@ -82,9 +90,13 @@ def with_bearer_token(bearer_token)
         # Send a request to the API and return the results
         #
         # Sends a request to the 
-        def request(method, path, body, options={})
-            
+        def request(method, path, body, params)
```

Comment:
use optional params or a default  values for body and params so that u dont have to pass nil while calling